pallets/oracle/src/lib.rs

```diff
@@ -237,19 +242,33 @@ impl<T: Config> Pallet<T> {
 	) -> DispatchResult {
 		let mut oracle_keys: Vec<_> = <OracleKeys<T>>::get();
 
-		for (k, v) in values {
+		for (key, value) in values {
 			let timestamped =
-				orml_oracle::TimestampedValue { timestamp: Self::get_current_time(), value: v };
-			T::DataFeedProvider::feed_value(oracle.clone(), k.clone(), timestamped)
+				orml_oracle::TimestampedValue { timestamp: Self::get_current_time(), value };
+			T::DataFeeder::feed_value(oracle.clone(), key.clone(), timestamped)
 				.expect("Expect store value by key");
-			if !oracle_keys.contains(&k) {
-				oracle_keys.push(k);
+			if !oracle_keys.contains(&key) {
+				oracle_keys.push(key);
 			}
 		}
 		<OracleKeys<T>>::put(oracle_keys.clone());
 		Ok(())
 	}
 
+	// public only for testing purposes
+	#[cfg(feature = "testing-utils")]
+	pub fn _clear_values() -> DispatchResult {
```

Comment:
Do we have a rule about when to use underscore prefix function names? 
Currently I had this impression of using such function/method _only_ internally... and I mean only on the _same lib/mod_. However, the `_clear_values()` is liberally called in other pallets like `issue` and `vault-registry`.

examples:
* the `fn _offchain_worker()` in `pallet/vault-registry` 
* the `fn _withdraw_collateral()` in `pallet/nomination`
*  the `fn _request_issue()` in `pallet/issue`

Reply:
Good point. We did not discuss this yet and I just used the underscore prefix because it is used by the other 'testing-utils' functions. But you are right that it's used in other pallets as well. 

Soooo... @pendulum-chain/devs what's your opinion on `_function()` names? I would use them for functions that are supposed to be used 'module-only' or helper functions. Thus, I would need to change the naming to `clear_values()` in this case.

Reply:
I now renamed all three functions because all of them are used in other pallets as well.